roles/ci_gen_kustomize_values/templates/nfv/edpm-values/values.yaml.j2

```diff
@@ -0,0 +1,61 @@
+{% set instances_names = []                                                            %}
+{% set _original_nodeset = (original_content.data | default({})).nodeset | default({}) %}
+{% set _original_nodes = _original_nodeset.nodes | default({})                         %}
+{% set _vm_type = (_original_nodes.keys() | first).split('-')[1]                       %}
+{% for _inst in cifmw_networking_env_definition.instances.keys()                       %}
+{%   if _inst.startswith(_vm_type)                                                     %}
+{%     set _ = instances_names.append(_inst)                                           %}
+{%   endif                                                                             %}
+{% endfor                                                                              %}
+data:
+  preProvisioned: false
+  baremetalSetTemplate:
+{% for instance in instances_names                                                     %}
+    ctlplaneInterface: "{{ cifmw_networking_env_definition.instances[instance].networks.ctlplane.mac_addr }}"
+    cloudUserName: cloud-admin
+    provisioningInterface: ## TODO!
```

Comment:
@pablintino @cjeanner i guess we need this value somehow. How we should we call it?

Reply:
I had discussed this with @pablintino earlier.  There were several ideas, but I think the last one we wanted to try was:

1. Pick a compute node and its MAC
2. Look at its respective `BaremetalHost` record and examine the `status.hardware.nics` and find the entry that matches the MAC
3. This entry will then have the interface name
4. Also, it is assumed all computes have the same hardware and interface names, so this discovered interface name can now be used for `baremetalSetTemplate.provisioningInterface`

Reply:
Oh sorry, I totally misread this and thought it involved `ctlplaneInterface`.  Ignore my comment :)

Reply:
Nice, cause in that case, as we set it in the bmh, it will pick it from there. We fill the bmh from a file that holds all the information for a given env, and there, apart of other fields, we set the macaddress of the provisioning interface, so I hope metal3 is able to find it properly.